estuary_updater/handlers/distgit.py

```diff
@@ -135,7 +135,7 @@ def parse_bugzilla_bugs(commit_message):
         """
         # Look for 'Resolves', 'Related', or 'Reverts' action Bugzilla bugs
         bugzilla_bug_pattern = re.compile(
-            r'(reverted:|resolves:|related:)([rhbz#, \d]+)')
+            r'(reverted:|resolves:|related:)(([rhbz#, ]*\d+)+)')
```

Comment:
After thinking about this, you could change `(reverted:|resolves:|related:)` to `(?:(reverted|resolves|related)\: *)`.

This would match it without the colon.